extensions/interactions/NumericInput/directives/numeric-input-validation.service.ts

```diff
@@ -163,6 +163,42 @@ export class NumericInputValidationService {
 
     return warningsList;
   }
+
+  getErrorString(value: string): string {
+    value = value.trim();
+    var trailingDot = /\.\d/g;
+    var twoDecimals = /.*\..*\./g;
+    var extraChars = /[^0-9.%+-]/g;
+    var trailingHyphen = /^-/g;
+    var extraHyphen = /-.*-/g;
+    if (!value) {
+      return '';
+    }
+    if (value.includes('.') && !value.match(trailingDot)) {
+      return 'Trailing decimals not allowed.';
+    } else if (value.match(twoDecimals)) {
```

Comment:
break after the opening ( here and below.

Reply:
Why? What is the reason behind this?

Reply:
Hey, @nithusha21 I have made the change you suggested but I'd like to know why? 
Isn't it better when it is in the same line?